schemas/webpackOptionsSchema.json

```diff
@@ -1,6 +1,34 @@
 {
   "additionalProperties": false,
   "definitions": {
+    "common.pluginFunction": {
+      "description": "Compiler or Resolver Plugin instance of Function",
+      "instanceof": "Function",
+      "properties": {
```

Comment:
I guess this is not needed as `apply` is inherited from the `Function.prototype`.

Reply:
this is needed or providing `{}` or `new function() {}` starts passing validation